src/CoreWCF.Templates/src/templates/CoreWCFService/Properties/launchSettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "profiles": {
+    "CoreWCFService": {
+      "commandName": "Project",
+      "launchBrowser": true,
```

Comment:
IMHO this line should be removed. We don't have the fancy test page that used to exist for soap services. The browser comes up saying it can't find a url, and it controls the lifetime of the process - so closing the browser stops debugging. Without it the process stays running and you can then test with a client, query for wsdl etc.

Reply:
i dropped the launchBrowser